jinja2/qfdmo/_addresses_partials/filters/carte_apropos.html

```diff
@@ -0,0 +1,68 @@
+<!-- Panel «Légende» -->
```

Comment:
C'est utile d'avoir ça dans le html en sortie ? 
On pourrait utiliser les commentaires jinja plutôt 
```suggestion
{# Panel «Légende» #}
```

Reply:
Fait pour tous les commentaires